<!DOCTYPE html>
<!-- Generated by pkgdown: do not edit by hand --><html>
<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1.0">
<title>directRNAExplorer - functionalities • directRNAExplorer</title>
<!-- jquery --><script src="https://code.jquery.com/jquery-3.1.0.min.js" integrity="sha384-nrOSfDHtoPMzJHjVTdCopGqIqeYETSXhZDFyniQ8ZHcVy08QesyHcnOUpMpqnmWq" crossorigin="anonymous"></script><!-- Bootstrap --><link href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous">
<script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js" integrity="sha384-Tc5IQib027qvyjSMfHjOMaLkfuWVxZxUPnCJA7l2mCWNIpG9mGCD8wGNIcPD7Txa" crossorigin="anonymous"></script><!-- Font Awesome icons --><link href="https://maxcdn.bootstrapcdn.com/font-awesome/4.6.3/css/font-awesome.min.css" rel="stylesheet" integrity="sha384-T8Gy5hrqNKT+hzMclPo118YTQO6cYprQmhrYwIiQ/3axmI1hQomh7Ud2hPOy8SP1" crossorigin="anonymous">
<!-- clipboard.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/1.7.1/clipboard.min.js" integrity="sha384-cV+rhyOuRHc9Ub/91rihWcGmMmCXDeksTtCihMupQHSsi8GIIRDG0ThDc3HGQFJ3" crossorigin="anonymous"></script><!-- pkgdown --><link href="../pkgdown.css" rel="stylesheet">
<script src="../jquery.sticky-kit.min.js"></script><script src="../pkgdown.js"></script><meta property="og:title" content="directRNAExplorer - functionalities">
<meta property="og:description" content="">
<meta name="twitter:card" content="summary">
<!-- mathjax --><script src="https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script><!--[if lt IE 9]>
<script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
<script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
<![endif]-->
</head>
<body>
    <div class="container template-article">
      <header><div class="navbar navbar-default navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <span class="navbar-brand">
        <a class="navbar-link" href="../index.html">directRNAExplorer</a>
        <span class="label label-default" data-toggle="tooltip" data-placement="bottom" title="Released package">0.0.0.9000</span>
      </span>
    </div>

    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
<li>
  <a href="../index.html">
    <span class="fa fa-home fa-lg"></span>
     
  </a>
</li>
<li>
  <a href="../reference/index.html">Reference</a>
</li>
<li class="dropdown">
  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">
    Articles
     
    <span class="caret"></span>
  </a>
  <ul class="dropdown-menu" role="menu">
<li>
      <a href="../articles/Package_presentation.html">directRNAExplorer - functionalities</a>
    </li>
    <li>
      <a href="../articles/testEdger.html">Likelihood ratio test and quasi-likelihood F-test for detecting changed genes</a>
    </li>
  </ul>
</li>
      </ul>
<ul class="nav navbar-nav navbar-right"></ul>
</div>
<!--/.nav-collapse -->
  </div>
<!--/.container -->
</div>
<!--/.navbar -->

      
      </header><div class="row">
  <div class="col-md-9 contents">
    <div class="page-header toc-ignore">
      <h1>directRNAExplorer - functionalities</h1>
                        <h4 class="author">Aleksandra Grudziaz</h4>
            
            <h4 class="date">2018-04-20</h4>
      
      
      <div class="hidden name"><code>Package_presentation.Rmd</code></div>

    </div>

    
    
<div id="bam-to--rda-conversion" class="section level2">
<h2 class="hasAnchor">
<a href="#bam-to--rda-conversion" class="anchor"></a>.bam to .rda conversion</h2>
<p>For conversion from .bam format to .rda we can use the <code><a href="../reference/bamToR.html">bamToR()</a></code> function. This function uses a Rsamtools package from the Bioconductor, so we have to load this package before.</p>
<div class="sourceCode"><pre class="sourceCode r"><code class="sourceCode r"><span class="kw">library</span>(Rsamtools)
<span class="kw">library</span>(directRNAExplorer)</code></pre></div>
<div class="sourceCode"><pre class="sourceCode r"><code class="sourceCode r">bamData &lt;-<span class="st"> </span><span class="kw"><a href="../reference/bamToR.html">bamToR</a></span>(<span class="st">"P13.2016-11-17T16_40_36.1234062861011.fc1.ch1.sorted.bam"</span>)</code></pre></div>
<p>We have to remember that we should set path to a directory containing this bam file.</p>
</div>
<div id="counts-per-gene" class="section level2">
<h2 class="hasAnchor">
<a href="#counts-per-gene" class="anchor"></a>Counts per gene</h2>
<p>Using frame created above we can find out, for each gene, how many nucleotides were recorded during the sequencing. For genes aggrergation we use the <code>TAIR10_genes</code> data. In my examples I use data for first chromosome.</p>
<div class="sourceCode"><pre class="sourceCode r"><code class="sourceCode r">datawt &lt;-<span class="st"> </span>dataChromosome1
databrm &lt;-<span class="st"> </span>brmDataChromosome1</code></pre></div>
<div class="sourceCode"><pre class="sourceCode r"><code class="sourceCode r"><span class="kw">head</span>(datawt)
<span class="co">#&gt; DataFrame with 6 rows and 13 columns</span>
<span class="co">#&gt;            qname      flag    rname   strand       pos    qwidth      mapq</span>
<span class="co">#&gt;      &lt;character&gt; &lt;integer&gt; &lt;factor&gt; &lt;factor&gt; &lt;integer&gt; &lt;integer&gt; &lt;integer&gt;</span>
<span class="co">#&gt; 1 70661909252085        16        1        -   2001464        21       255</span>
<span class="co">#&gt; 2 70646155445083        16        1        -   2001594        23       255</span>
<span class="co">#&gt; 3 70657379404766        16        1        -   2002661        29       255</span>
<span class="co">#&gt; 4 70643789860258        16        1        -   2002663        26       255</span>
<span class="co">#&gt; 5 70652883110414        16        1        -   2002664        25       255</span>
<span class="co">#&gt; 6 70657379411167        16        1        -   2002666        24       255</span>
<span class="co">#&gt;             cigar     mrnm      mpos     isize                     seq</span>
<span class="co">#&gt;       &lt;character&gt; &lt;factor&gt; &lt;integer&gt; &lt;integer&gt;          &lt;DNAStringSet&gt;</span>
<span class="co">#&gt; 1     1M1D13M1D7M       NA         0         0   GTTATCTTTTTTTTTTTATCT</span>
<span class="co">#&gt; 2     6M1D1M2D16M       NA         0         0 TTTCTGTTTTTTCTCGCTGTTGC</span>
<span class="co">#&gt; 3         1M1D28M       NA         0         0 TTTATGTGCT...GATTGGTTTC</span>
<span class="co">#&gt; 4 16M2D5M1D4M1D1M       NA         0         0 TTATGTGCTT...GATTGTTTCC</span>
<span class="co">#&gt; 5         1M1D24M       NA         0         0 TTGTGCTTGA...TGATTGGTTT</span>
<span class="co">#&gt; 6             24M       NA         0         0 TGTGCTTGAG...TGATTGGTTT</span>
<span class="co">#&gt;                      qual</span>
<span class="co">#&gt;            &lt;PhredQuality&gt;</span>
<span class="co">#&gt; 1   .....................</span>
<span class="co">#&gt; 2 .......................</span>
<span class="co">#&gt; 3 .......................</span>
<span class="co">#&gt; 4 .......................</span>
<span class="co">#&gt; 5 .......................</span>
<span class="co">#&gt; 6 .......................</span></code></pre></div>
<div class="sourceCode"><pre class="sourceCode r"><code class="sourceCode r">genesSummary &lt;-<span class="st"> </span><span class="kw"><a href="../reference/genesSummarize.html">genesSummarize</a></span>(datawt, <span class="dt">chromosome =</span> <span class="dv">1</span>)
genesSummary &lt;-<span class="st"> </span>dplyr<span class="op">::</span><span class="kw"><a href="http://dplyr.tidyverse.org/reference/arrange.html">arrange</a></span>(genesSummary, <span class="kw">desc</span>(counts))

<span class="kw">head</span>(genesSummary)
<span class="co">#&gt;        name counts uniqueCounts length</span>
<span class="co">#&gt; 1 AT1G01010      0            0   2268</span>
<span class="co">#&gt; 2 AT1G01020      0            0   2809</span>
<span class="co">#&gt; 3 AT1G01030      0            0   2065</span>
<span class="co">#&gt; 4 AT1G01040      0            0   8081</span>
<span class="co">#&gt; 5 AT1G01046      0            0    206</span>
<span class="co">#&gt; 6 AT1G01050      0            0   1983</span></code></pre></div>
</div>
<div id="comparision-between-two-groups" class="section level2">
<h2 class="hasAnchor">
<a href="#comparision-between-two-groups" class="anchor"></a>Comparision between two groups</h2>
<p>We can compare the distribution of counts between two genes or part of the gene (for example three prime UTR, exons). We also can compare distributions between positive or negative strands. We can use for this comparision for example the two-sample Kolmogorov-Smirnov test. In next example I will use the data for gene “AT1G06680”.</p>
<div class="sourceCode"><pre class="sourceCode r"><code class="sourceCode r"><span class="kw"><a href="../reference/ksDistributionTest.html">ksDistributionTest</a></span>(datawt, databrm, <span class="st">"AT1G06760"</span>)
<span class="co">#&gt; </span>
<span class="co">#&gt;  Two-sample Kolmogorov-Smirnov test</span>
<span class="co">#&gt; </span>
<span class="co">#&gt; data:  dt_neg[dt_neg$condition == "type1", "position"] and dt_neg[dt_neg$condition == "type2", "position"]</span>
<span class="co">#&gt; D = 0.079927, p-value = 0.9829</span>
<span class="co">#&gt; alternative hypothesis: two-sided</span></code></pre></div>
<p>For selected parts of chromosomes in two groups we can compute the table with p-values for chosen test about their distributions.</p>
<div class="sourceCode"><pre class="sourceCode r"><code class="sourceCode r">table &lt;-<span class="st"> </span><span class="kw"><a href="../reference/pvalTable.html">pvalTable</a></span>(datawt, databrm, <span class="dt">chromosome=</span><span class="st">"1"</span>, <span class="dt">type =</span><span class="st">"KS"</span>)
<span class="kw">head</span>(dplyr<span class="op">::</span><span class="kw"><a href="http://dplyr.tidyverse.org/reference/arrange.html">arrange</a></span>(table, pval),<span class="dv">6</span>)
<span class="co">#&gt;        gene       pval  statistic</span>
<span class="co">#&gt; 1 AT1G06720 0.02736573 0.45174825</span>
<span class="co">#&gt; 2 AT1G06630 0.07844748 0.40079365</span>
<span class="co">#&gt; 3 AT1G06590 0.08502377 0.50000000</span>
<span class="co">#&gt; 4 AT1G06730 0.16083539 0.50000000</span>
<span class="co">#&gt; 5 AT1G06690 0.17340708 0.19909688</span>
<span class="co">#&gt; 6 AT1G06680 0.24452875 0.03018483</span></code></pre></div>
</div>
<div id="coverage-plot" class="section level2">
<h2 class="hasAnchor">
<a href="#coverage-plot" class="anchor"></a>Coverage plot</h2>
<p>For chosen gene we can draw a coverage plot for <code>datawt</code> and <code>databrm</code>. First we have plots for the three prime UTR.</p>
<div class="sourceCode"><pre class="sourceCode r"><code class="sourceCode r"><span class="kw"><a href="../reference/plotGeneDistribution.html">plotGeneDistribution</a></span>(<span class="dt">gene=</span><span class="st">"AT1G06760"</span>, <span class="dt">bamDataFrame=</span>datawt)</code></pre></div>
<p><img src="Package_presentation_files/figure-html/unnamed-chunk-8-1.png" width="672"></p>
<div class="sourceCode"><pre class="sourceCode r"><code class="sourceCode r"><span class="kw"><a href="../reference/plotGeneDistribution.html">plotGeneDistribution</a></span>(<span class="dt">gene=</span><span class="st">"AT1G06760"</span>, <span class="dt">bamDataFrame=</span>databrm)</code></pre></div>
<p><img src="Package_presentation_files/figure-html/unnamed-chunk-8-2.png" width="672"></p>
<p>And plots for whole gene.</p>
<div class="sourceCode"><pre class="sourceCode r"><code class="sourceCode r"><span class="kw"><a href="../reference/plotGeneDistribution.html">plotGeneDistribution</a></span>(<span class="dt">gene=</span><span class="st">"AT1G06760"</span>, <span class="dt">bamDataFrame=</span>datawt)</code></pre></div>
<p><img src="Package_presentation_files/figure-html/unnamed-chunk-9-1.png" width="672"></p>
<div class="sourceCode"><pre class="sourceCode r"><code class="sourceCode r"><span class="kw"><a href="../reference/plotGeneDistribution.html">plotGeneDistribution</a></span>(<span class="dt">gene=</span><span class="st">"AT1G06760"</span>, <span class="dt">bamDataFrame=</span>databrm)</code></pre></div>
<p><img src="Package_presentation_files/figure-html/unnamed-chunk-9-2.png" width="672"></p>
<p>We can also see the distribution of these two samples on one plot.</p>
</div>
<div id="r-fig-width7-plotgenecomparisondistributionat1g06760-datawt-databrm" class="section level1">
<h1 class="hasAnchor">
<a href="#r-fig-width7-plotgenecomparisondistributionat1g06760-datawt-databrm" class="anchor"></a><code>{r, fig.width=7} # plotGeneComparisonDistribution("AT1G06760", datawt, databrm) #</code>
</h1>
<p>In this case red color lines correspond to data from first dataset and blue - to second.</p>
<p>We can also see the ecdf of these two samples on one plot.</p>
<div class="sourceCode"><pre class="sourceCode r"><code class="sourceCode r"><span class="kw"><a href="../reference/plotGeneComparisonDistribution.html">plotGeneComparisonDistribution</a></span>(<span class="st">"AT1G06760"</span>, datawt, databrm,<span class="dt">stat=</span><span class="st">"ecdf"</span>)</code></pre></div>
<p><img src="Package_presentation_files/figure-html/unnamed-chunk-10-1.png" width="672"></p>
<div id="testing-counts-in-genes" class="section level2">
<h2 class="hasAnchor">
<a href="#testing-counts-in-genes" class="anchor"></a>Testing counts in genes</h2>
<p>For two types of individuals we can compute negative binomial test to find significantly changed genes between one and the other type.</p>
<div id="table-of-counts" class="section level3">
<h3 class="hasAnchor">
<a href="#table-of-counts" class="anchor"></a>Table of counts</h3>
<p>Firstly, we have to compute, from sequencing data, the table of counts for each gene from Arabidopsis. We can create this table using <code>countsForAllGenes</code> function.</p>
<div class="sourceCode"><pre class="sourceCode r"><code class="sourceCode r">counts_genes_wt1 &lt;-<span class="st"> </span><span class="kw"><a href="../reference/countsForAllGenes.html">countsForAllGenes</a></span>(dataChromosome1, <span class="dt">geneData =</span> directRNAExplorer<span class="op">::</span>Araport11, <span class="dt">genePart=</span><span class="st">"gene"</span>)</code></pre></div>
</div>
<div id="testing" class="section level3">
<h3 class="hasAnchor">
<a href="#testing" class="anchor"></a>Testing</h3>
<p>To test differences between types of individuals we must compute at least two counts tables. One for the first type, and then for the second type. I’ve create a table contains inforamation for two types - wild type and a mutant type (brm). We have seven samples in our test.</p>
<div class="sourceCode"><pre class="sourceCode r"><code class="sourceCode r">allCounts &lt;-<span class="st"> </span>directRNAExplorer<span class="op">::</span>countsAll</code></pre></div>
</div>
<div id="choice-of-counts" class="section level3">
<h3 class="hasAnchor">
<a href="#choice-of-counts" class="anchor"></a>Choice of counts</h3>
<p>We want to test these genes which could be significant. Firstly, we should select genes which have “good number” of counts. We assumed that the counts are “good” if more than at least 2 samples in the wt case and at least 3 samples in brm case we have more than 9 counts for gene. In addition, we are looking for such genes that have many counts in one of the subtypes, and almost zero in the second one.</p>
<div class="sourceCode"><pre class="sourceCode r"><code class="sourceCode r">genesTest &lt;-<span class="kw">c</span>() 
countsWithCondition &lt;-<span class="st"> </span>allCounts


<span class="cf">for</span>(i <span class="cf">in</span> <span class="dv">2</span><span class="op">:</span><span class="kw">ncol</span>(allCounts)){
  countsWithCondition[,i] &lt;-<span class="st"> </span><span class="kw">ifelse</span>(allCounts[,i]<span class="op">&gt;=</span><span class="dv">9</span>, <span class="dv">1</span>, <span class="dv">0</span>)
}

allCounts_<span class="dv">2</span> &lt;-<span class="st"> </span>countsWithCondition
genes &lt;-<span class="st"> </span>allCounts_<span class="dv">2</span>[,<span class="dv">1</span>]

allCountsTransposed &lt;-<span class="st"> </span><span class="kw">t</span>(allCounts_<span class="dv">2</span>[,<span class="op">-</span><span class="dv">1</span>])
allCountsTransposed &lt;-<span class="st"> </span><span class="kw">data.frame</span>(allCountsTransposed)
<span class="kw">colnames</span>(allCountsTransposed)&lt;-<span class="st"> </span>genes

allCountsTransposed<span class="op">$</span>type &lt;-<span class="st"> </span><span class="kw">c</span>(<span class="kw">rep</span>(<span class="st">"wt"</span>, <span class="dv">3</span>), <span class="kw">rep</span>(<span class="st">"c"</span>, <span class="dv">4</span>))
allCountsTransposed &lt;-<span class="st"> </span>allCountsTransposed[,<span class="kw">c</span>(<span class="kw">ncol</span>(allCountsTransposed), <span class="dv">1</span><span class="op">:</span>(<span class="kw">ncol</span>(allCountsTransposed)<span class="op">-</span><span class="dv">1</span>))]

sums &lt;-<span class="st"> </span><span class="kw">aggregate</span>(. <span class="op">~</span><span class="st"> </span>type, <span class="dt">data=</span>allCountsTransposed, <span class="dt">FUN=</span>sum)

<span class="cf">for</span>(i <span class="cf">in</span> <span class="dv">2</span><span class="op">:</span><span class="kw">ncol</span>(allCountsTransposed)){
  <span class="cf">if</span>(((sums[<span class="dv">1</span>,i]<span class="op">&gt;=</span><span class="dv">3</span>) <span class="op">&amp;</span><span class="st"> </span>(sums[<span class="dv">2</span>,i]<span class="op">&gt;=</span><span class="dv">2</span>)) <span class="op">||</span><span class="st"> </span>((sums[<span class="dv">1</span>,i]<span class="op">&gt;=</span><span class="dv">3</span>) <span class="op">&amp;</span><span class="st"> </span>(sums[<span class="dv">2</span>,i]<span class="op">==</span><span class="dv">0</span>))<span class="op">||</span>((sums[<span class="dv">1</span>,i]<span class="op">==</span><span class="dv">0</span>) <span class="op">&amp;</span><span class="st"> </span>(sums[<span class="dv">2</span>,i]<span class="op">&gt;=</span><span class="dv">2</span>))){
    genesTest[i] &lt;-<span class="st"> </span><span class="kw">colnames</span>(allCountsTransposed)[i] 
  }<span class="cf">else</span>{
    genesTest[i] &lt;-<span class="st"> </span><span class="ot">NA</span>
  }
}

genesTest2&lt;-<span class="st"> </span><span class="kw">na.omit</span>(genesTest)

countsTest &lt;-<span class="st"> </span>allCounts
countsTest &lt;-<span class="st"> </span>countsTest[<span class="kw">which</span>(countsTest<span class="op">$</span>name <span class="op">%in%</span><span class="st"> </span>genesTest2),]

countsTest2 &lt;-<span class="st"> </span><span class="kw">data.frame</span>(<span class="kw">t</span>(countsTest[,<span class="op">-</span><span class="dv">1</span>]))
<span class="kw">colnames</span>(countsTest2) &lt;-<span class="st"> </span>countsTest<span class="op">$</span>name</code></pre></div>
</div>
<div id="nbinom-test" class="section level3">
<h3 class="hasAnchor">
<a href="#nbinom-test" class="anchor"></a>nbinom test</h3>
<p>For selected genes we perform a binomial negative test, in order to find genes for which the number of readings has changed significantly divided into two groups - wt and brm.</p>
<div class="sourceCode"><pre class="sourceCode r"><code class="sourceCode r">cond  &lt;-<span class="st"> </span><span class="kw">c</span>(<span class="kw">rep</span>(<span class="st">"wt"</span>, <span class="dv">3</span>), <span class="kw">rep</span>(<span class="st">"c"</span>, <span class="dv">4</span>))
test&lt;-<span class="kw"><a href="../reference/nbinomTest.html">nbinomTest</a></span>(countsTest2, <span class="dt">condition =</span> cond)</code></pre></div>
<p>To find the most changed genes, we will focus only on those that have a small pvalue and fold greater than 1.5.</p>
<div class="sourceCode"><pre class="sourceCode r"><code class="sourceCode r"><span class="kw">library</span>(dplyr)
testFiltered &lt;-<span class="st"> </span><span class="kw">filter</span>(test, pvalue<span class="op">&lt;</span><span class="fl">0.05</span>)
testFiltered &lt;-<span class="st"> </span><span class="kw">filter</span>(testFiltered, <span class="kw">abs</span>(log2FoldChange)<span class="op">&gt;</span><span class="fl">0.6</span>)
testFiltered &lt;-<span class="st"> </span><span class="kw">arrange</span>(testFiltered, pvalue)
testFiltered &lt;-<span class="st"> </span><span class="kw">arrange</span>(testFiltered, <span class="kw">desc</span>(log2FoldChange))

<span class="kw">head</span>(testFiltered)
<span class="co">#&gt;     baseMean log2FoldChange     lfcSE      stat        pvalue</span>
<span class="co">#&gt; 1  33.386784       4.166556 0.4017215 10.371754  3.333487e-25</span>
<span class="co">#&gt; 2   6.596427       3.058755 0.6610824  4.626889  3.712002e-06</span>
<span class="co">#&gt; 3  18.217809       3.037973 0.4748173  6.398193  1.572260e-10</span>
<span class="co">#&gt; 4 446.422432       2.889503 0.1256007 23.005469 4.109286e-117</span>
<span class="co">#&gt; 5  40.056709       2.883768 0.2812684 10.252726  1.150523e-24</span>
<span class="co">#&gt; 6  26.215771       2.514303 0.3960365  6.348666  2.171903e-10</span>
<span class="co">#&gt;            padj        id</span>
<span class="co">#&gt; 1  2.804296e-22 AT3G01345</span>
<span class="co">#&gt; 2  2.342041e-04 AT1G47580</span>
<span class="co">#&gt; 3  2.834280e-08 AT1G53480</span>
<span class="co">#&gt; 4 4.148324e-113 AT2G42540</span>
<span class="co">#&gt; 5  8.934253e-22 AT1G22690</span>
<span class="co">#&gt; 6  3.716164e-08 AT4G15490</span></code></pre></div>
</div>
</div>
</div>
  </div>

  <div class="col-md-3 hidden-xs hidden-sm" id="sidebar">
        <div id="tocnav">
      <h2 class="hasAnchor">
<a href="#tocnav" class="anchor"></a>Contents</h2>
      <ul class="nav nav-pills nav-stacked">
<li><a href="#bam-to--rda-conversion">.bam to .rda conversion</a></li>
      <li><a href="#counts-per-gene">Counts per gene</a></li>
      <li><a href="#comparision-between-two-groups">Comparision between two groups</a></li>
      <li><a href="#coverage-plot">Coverage plot</a></li>
      <li>
<a href="#r-fig-width7-plotgenecomparisondistributionat1g06760-datawt-databrm"><code>{r, fig.width=7} # plotGeneComparisonDistribution("AT1G06760", datawt, databrm) #</code></a><ul class="nav nav-pills nav-stacked">
<li><a href="#testing-counts-in-genes">Testing counts in genes</a></li>
      </ul>
</li>
      </ul>
</div>
      </div>

</div>


      <footer><div class="copyright">
  <p>Developed by Aleksandra Grudziaz.</p>
</div>

<div class="pkgdown">
  <p>Site built with <a href="http://pkgdown.r-lib.org/">pkgdown</a>.</p>
</div>

      </footer>
</div>

  

  </body>
</html>
